service_info/templates/aldryn_newsblog/article_detail.html

```diff
@@ -12,20 +12,27 @@
 {% endblock %}
 
 {% block newsblog_footer %}
-    <div class="aldryn-newsblog-pager">
-        <ul class="pager pagination">
-            {% if prev_article %}
-                <li class="previous"><a href="{{ prev_article.get_absolute_url }}"><span aria-hidden="true">&larr;</span> {% trans "Previous Article" %}</a></li>
-            {% else %}
-                <li class="previous inactive"><span class="text-muted"><span aria-hidden="true">&larr;</span> {% trans "Previous Article" %}</span></li>
-            {% endif %}
-            <li><span class="pager-back"><a href="{% namespace_url "article-list" %}">{% trans "Back to Overview" %}</a></span></li>
-            {% if next_article %}
-                <li class="next"><a href="{{ next_article.get_absolute_url }}">{% trans "Next Article" %} <span aria-hidden="true">&rarr;</span></a></li>
-            {% else %}
-                <li class="next inactive"><span class="text-muted">{% trans "Next Article" %} <span aria-hidden="true">&rarr;</span></span></li>
-            {% endif %}
-        </ul>
-    </div>
     {% render_placeholder view.config.placeholder_detail_footer %}
+
+    <nav class="aldryn-newsblog-pager grey darken-1">
+        <div class="nav-wrapper">
+            <ul class="pager">
+                {% if prev_article %}
+                    <li class="previous"><a href="{{ prev_article.get_absolute_url }}">{% trans "Previous" %}</a></li>
+                {% else %}
+                    <li class="previous">{% trans "Previous" %}</li>
```

Comment:
Actually, that's just a remnant of old implementations. It doesn't do anything, nor is there any way to make it do what I was trying to do. Instead I am just going to remove the link altogether when it wouldn't do anything.